pkg/devfile/generator/utils.go

```diff
@@ -197,7 +198,7 @@ func getServiceSpec(devfileObj parser.DevfileObj, selectorLabels map[string]stri
 
 	var containerPorts []corev1.ContainerPort
 	portExposureMap := getPortExposure(devfileObj)
-	containers, err := GetContainers(devfileObj)
+	containers, err := GetContainers(devfileObj, common.DevfileOptions{})
```

Comment:
should we add `DevfileOptions` to getService as well? to filter out endpoints from specific components / with specific attributes for service creation

Reply:
updated to incl options to service but endpoint will be discussed and added later if necessary since its a level 2 attribute